packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -157,15 +158,15 @@ export class Meter implements api.Meter {
   }
 
   /**
-   * Creates a new value observer metric.
+   * Creates a new `ValueObserver` metric.
```

Comment:
```suggestion
   * Creates a new `ValueObserver` instrument.
```

I think this is an instrument rather than a metric?